x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/application/EnterpriseSearchFeatureSetUsageTests.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.core.application;
+
+import org.elasticsearch.common.io.stream.Writeable;
+import org.elasticsearch.test.AbstractWireSerializingTestCase;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class EnterpriseSearchFeatureSetUsageTests extends AbstractWireSerializingTestCase<EnterpriseSearchFeatureSetUsage> {
```

Comment:
Nitpick: I think we might want to rename this to `EnterpriseSearchFeatureSetUsageSerializingTests` - that matches the name we've given to other serialization tests and also keeps `EnterpriseSeearchFeatureSetUsageTests` free for any future tests we may want to add to the usage object itself.

Reply:
✅ https://github.com/elastic/elasticsearch/pull/95306/commits/85e0913b3f32b86bfd852698e24ea646ee28bbac